CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 ### PDFOut - Changelog
 ___
 
+### 05.01.2019 Version 1.6.0
+- Wenn man hinter der URL `?pdftest=1` eingibt, kommt zum test die README des Addons.
```

Comment:
Ich würde in der readme des addons irgendwo nen link/button en „print as pdf“ o.ä. Sodas es nicht so ein geheimes feature vleibt

Reply:
Wie genau meinst du das.

Reply:
Wenn man die readme des addon sich im backend betrachtet, würde ich dort einen button anzeigen. Dieser kann eure test url tragen beim klick würde er dann die readme als pdf an den browser schicken

Reply:
Wie schreibt man einen Button mit MD?, und wie müsste das dann mit der URL Aussehen.. weil es muss ja dann noch die Haupturl ausgegeben werden und die ist ja nicht immer gleich.. Oder verstehe ich gerade etwas Falsch?.

Reply:
Am besten die spielst mal ein bisschen mit rum und probierst mal was aus.
Nur so kann man lernen wann was sinnvoll ist

Reply:
Das mit dem Button würde ich eventuell noch hinkriegen, nur mit der Url wird es dann glaub wieder etwas Komplizierter.

Reply:
Weniger vermuten - mehr mut zum ausprobieren :-)

Reply:
Hmm

Reply:
Markus hat Recht. Du musst dich nun einarbeiten. Versuche anhand der anderen AddOns und der Doku zu lernen.